src/ocean/text/convert/DateTime_tango.d

```diff
@@ -586,11 +604,9 @@ struct DateTimeLocale
         return f;
     }
 
-    /**********************************************************************
-
-     **********************************************************************/
-
-    private char[] formatCustom (ref Result result, Time dateTime, cstring format)
+    /// Implementation of format
+    private void formatCustom (size_t delegate(cstring) sink, Time dateTime,
```

Comment:
Suggest splitting this change to `formatCustom` to accept a sink delegate into a separate commit. It's not clear to me where the "Format Time correctly with Formatter" part comes in.